org.quartz.JobPersistenceException when running a PositionCalculation job

When running a _PositionCalculation_ job involving a position based on equities, an _org.quartz.JobPersistenceException_  exception is thrown.

Steps to reproduce the behavior:
1. Create a _PositionDefinition_ involving the _Equity_ product type. Ensure to have one live trade on equities.
2. Create a _PositionCalculation_ job using this position and run it
3. The job fails and there is this exception (see below). Note that there is no execution result in the result panel but only this exception on server side

```
08:34:07,482 ERROR [stderr] (default task-2) org.quartz.JobPersistenceException: Couldn't store trigger 'DEFAULT.MT_1nac6sd8plhbs' for 'AOC.PC1' job:Couldn't retrieve job because a required class was not found: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))] [See nested exception: org.quartz.JobPersistenceException: Couldn't retrieve job because a required class was not found: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))] [See nested exception: java.lang.ClassNotFoundException: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))]]]

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport.storeTrigger(JobStoreSupport.java:1225)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport$4.executeVoid(JobStoreSupport.java:1160)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport$VoidTransactionCallback.execute(JobStoreSupport.java:3716)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport$VoidTransactionCallback.execute(JobStoreSupport.java:3714)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreCMT.executeInLock(JobStoreCMT.java:242)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport.storeTrigger(JobStoreSupport.java:1156)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.core.QuartzScheduler.triggerJob(QuartzScheduler.java:1148)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.StdScheduler.triggerJob(StdScheduler.java:342)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.StdScheduler.triggerJob(StdScheduler.java:332)

08:34:07,482 ERROR [stderr] (default task-2) 	at deployment.app-3.1.0.ear.core-ejb-3.1.0.jar//org.eclipse.tradista.core.batch.service.BatchServiceBean.runJobInstance(BatchServiceBean.java:280)

08:34:07,482 ERROR [stderr] (default task-2) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)

08:34:07,482 ERROR [stderr] (default task-2) 	at java.base/java.lang.reflect.Method.invoke(Method.java:565)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.ManagedReferenceMethodInterceptor.processInvocation(ManagedReferenceMethodInterceptor.java:35)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@36.0.1.Final//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.delegateInterception(Jsr299BindingsInterceptor.java:62)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@36.0.1.Final//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.doMethodInterception(Jsr299BindingsInterceptor.java:72)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@36.0.1.Final//org.jboss.as.weld.interceptors.Jsr299BindingsInterceptor.processInvocation(Jsr299BindingsInterceptor.java:85)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.interceptors.UserInterceptorFactory$1.processInvocation(UserInterceptorFactory.java:46)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,482 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)

08:34:07,482 ERROR [stderr] (default task-2) 	at deployment.app-3.1.0.ear//org.eclipse.tradista.core.common.service.TradistaAuthorizationFilteringInterceptor.proceed(TradistaAuthorizationFilteringInterceptor.java:50)

08:34:07,482 ERROR [stderr] (default task-2) 	at deployment.app-3.1.0.ear//org.eclipse.tradista.core.batch.service.JobFilteringInterceptor.filter(JobFilteringInterceptor.java:43)

08:34:07,483 ERROR [stderr] (default task-2) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)

08:34:07,483 ERROR [stderr] (default task-2) 	at java.base/java.lang.reflect.Method.invoke(Method.java:565)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.ManagedReferenceLifecycleMethodInterceptor.processInvocation(ManagedReferenceLifecycleMethodInterceptor.java:72)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)

08:34:07,483 ERROR [stderr] (default task-2) 	at deployment.app-3.1.0.ear//org.eclipse.tradista.core.common.service.TradistaExceptionHandlerInterceptor.handleException(TradistaExceptionHandlerInterceptor.java:43)

08:34:07,483 ERROR [stderr] (default task-2) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)

08:34:07,483 ERROR [stderr] (default task-2) 	at java.base/java.lang.reflect.Method.invoke(Method.java:565)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.ManagedReferenceLifecycleMethodInterceptor.processInvocation(ManagedReferenceLifecycleMethodInterceptor.java:72)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.invocationmetrics.ExecutionTimeInterceptor.processInvocation(ExecutionTimeInterceptor.java:26)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.as.jpa@36.0.1.Final//org.jboss.as.jpa.interceptor.SBInvocationInterceptor.processInvocation(SBInvocationInterceptor.java:30)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.wildfly.concurrency@36.0.1.Final//org.jboss.as.ee.concurrent.ConcurrentContextInterceptor.processInvocation(ConcurrentContextInterceptor.java:28)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InitialInterceptor.processInvocation(InitialInterceptor.java:40)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:53)

08:34:07,483 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.interceptors.ComponentDispatcherInterceptor.processInvocation(ComponentDispatcherInterceptor.java:35)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.pool.PooledInstanceInterceptor.processInvocation(PooledInstanceInterceptor.java:34)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.interceptors.AdditionalSetupInterceptor.processInvocation(AdditionalSetupInterceptor.java:39)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.tx.CMTTxInterceptor.invokeInOurTx(CMTTxInterceptor.java:237)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.tx.CMTTxInterceptor.required(CMTTxInterceptor.java:373)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.tx.CMTTxInterceptor.processInvocation(CMTTxInterceptor.java:143)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext$Invocation.proceed(InterceptorContext.java:509)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.weld.core@5.1.5.Final//org.jboss.weld.module.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:87)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.weld.common@36.0.1.Final//org.jboss.as.weld.ejb.EjbRequestScopeActivationInterceptor.processInvocation(EjbRequestScopeActivationInterceptor.java:72)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.interceptors.CurrentInvocationContextInterceptor.processInvocation(CurrentInvocationContextInterceptor.java:24)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.invocationmetrics.WaitTimeInterceptor.processInvocation(WaitTimeInterceptor.java:30)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.security.IdentityOutflowInterceptor.processInvocation(IdentityOutflowInterceptor.java:56)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.security.SecurityDomainInterceptor.processInvocation(SecurityDomainInterceptor.java:27)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.deployment.processors.StartupAwaitInterceptor.processInvocation(StartupAwaitInterceptor.java:27)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.interceptors.ShutDownInterceptorFactory$1.processInvocation(ShutDownInterceptorFactory.java:47)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.deployment.processors.EjbSuspendInterceptor.processInvocation(EjbSuspendInterceptor.java:40)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.component.interceptors.LoggingInterceptor.processInvocation(LoggingInterceptor.java:50)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.NamespaceContextInterceptor.processInvocation(NamespaceContextInterceptor.java:33)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.ContextClassLoaderInterceptor.processInvocation(ContextClassLoaderInterceptor.java:60)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.run(InterceptorContext.java:438)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.wildfly.security.elytron-base@2.6.4.Final//org.wildfly.security.manager.WildFlySecurityManager.doChecked(WildFlySecurityManager.java:633)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.AccessCheckingInterceptor.processInvocation(AccessCheckingInterceptor.java:57)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.InterceptorContext.proceed(InterceptorContext.java:422)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.invocation@2.0.1.Final//org.jboss.invocation.ChainedInterceptor.processInvocation(ChainedInterceptor.java:53)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ee@36.0.1.Final//org.jboss.as.ee.component.ViewService$View.invoke(ViewService.java:181)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.wildfly.security.elytron-base@2.6.4.Final//org.wildfly.security.auth.server.SecurityIdentity.runAsFunctionEx(SecurityIdentity.java:421)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.remote.AssociationImpl.invokeWithIdentity(AssociationImpl.java:638)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.remote.AssociationImpl.invokeMethod(AssociationImpl.java:616)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.remote.AssociationImpl.lambda$receiveInvocationRequest$0(AssociationImpl.java:214)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.remote.AssociationImpl.execute(AssociationImpl.java:305)

08:34:07,484 ERROR [stderr] (default task-2) 	at org.jboss.as.ejb3@36.0.1.Final//org.jboss.as.ejb3.remote.AssociationImpl.receiveInvocationRequest(AssociationImpl.java:258)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.ejb-client@5.0.8.Final//org.jboss.ejb.protocol.remote.EJBServerChannel$ReceiverImpl.handleInvocationRequest(EJBServerChannel.java:477)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.ejb-client@5.0.8.Final//org.jboss.ejb.protocol.remote.EJBServerChannel$ReceiverImpl.handleMessage(EJBServerChannel.java:212)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.remoting@5.0.31.Final//org.jboss.remoting3.remote.RemoteConnectionChannel.lambda$handleMessageData$3(RemoteConnectionChannel.java:432)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.remoting@5.0.31.Final//org.jboss.remoting3.EndpointImpl$TrackingExecutor.lambda$execute$0(EndpointImpl.java:993)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.xnio@3.8.16.Final//org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282)

08:34:07,486 ERROR [stderr] (default task-2) 	at java.base/java.lang.Thread.run(Thread.java:1447)

08:34:07,486 ERROR [stderr] (default task-2) Caused by: org.quartz.JobPersistenceException: Couldn't retrieve job because a required class was not found: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))] [See nested exception: java.lang.ClassNotFoundException: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))]]

08:34:07,486 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveJob(JobStoreSupport.java:1408)

08:34:07,486 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport.storeTrigger(JobStoreSupport.java:1206)

08:34:07,486 ERROR [stderr] (default task-2) 	... 97 more

08:34:07,486 ERROR [stderr] (default task-2) Caused by: java.lang.ClassNotFoundException: org.eclipse.tradista.security.equity.model.Equity from [Module "org.eclipse.tradista.org.quartz-scheduler.quartz" version 2.5.2 from local module loader @49872d67 (finder: local module finder @56303b57 (roots: C:\Softwares\wildfly-36.0.1.Final\modules,C:\Softwares\wildfly-36.0.1.Final\modules\system\layers\base))]

08:34:07,486 ERROR [stderr] (default task-2) 	at org.jboss.modules.ModuleClassLoader.findClass(ModuleClassLoader.java:201)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.jboss.modules.ConcurrentClassLoader.performLoadClassUnchecked(ConcurrentClassLoader.java:408)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.jboss.modules.ConcurrentClassLoader.performLoadClass(ConcurrentClassLoader.java:396)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.jboss.modules.ConcurrentClassLoader.loadClass(ConcurrentClassLoader.java:115)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.lang.Class.forName0(Native Method)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.lang.Class.forName(Class.java:543)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.resolveClass(ObjectInputStream.java:753)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1945)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1818)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2134)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1653)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream$FieldValues.<init>(ObjectInputStream.java:2513)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2360)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2166)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1653)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:495)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:453)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.util.HashMap.readObject(HashMap.java:1561)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.lang.reflect.Method.invoke(Method.java:565)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectStreamClass.invokeReadObject(ObjectStreamClass.java:976)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2335)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2166)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1653)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream$FieldValues.<init>(ObjectInputStream.java:2513)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readSerialData(ObjectInputStream.java:2360)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:2166)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1653)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:495)

08:34:07,487 ERROR [stderr] (default task-2) 	at java.base/java.io.ObjectInputStream.readObject(ObjectInputStream.java:453)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.StdJDBCDelegate.getObjectFromBlob(StdJDBCDelegate.java:3424)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.StdJDBCDelegate.handleJobDetails(StdJDBCDelegate.java:1657)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectJobDetail(StdJDBCDelegate.java:844)

08:34:07,487 ERROR [stderr] (default task-2) 	at org.eclipse.tradista.org.quartz-scheduler.quartz@2.5.2//org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveJob(JobStoreSupport.java:1403)

08:34:07,488 ERROR [stderr] (default task-2) 	... 98 more
```

**Screenshots**

<img width="433" height="175" alt="Image" src="https://github.com/user-attachments/assets/bf0702d0-a95b-43a8-b561-5d7a3cbe5c2a" />

<img width="650" height="423" alt="Image" src="https://github.com/user-attachments/assets/e87c11b1-2228-469a-b828-de2f02fa2ef8" />


**Environment:**
 - Version 3.1.0

--- a/tradista-wildfly-setup/modules/org/eclipse/tradista/org/quartz-scheduler/quartz/main/module.xml
+++ b/tradista-wildfly-setup/modules/org/eclipse/tradista/org/quartz-scheduler/quartz/main/module.xml
@@ -8,7 +8,11 @@
 
 	<dependencies>
 		<module name="org.slf4j" />
-		<module name="org.eclipse.tradista.core.model" export="true" />
+		<module name="org.eclipse.tradista.fx.model" export="true" />
+		<module name="org.eclipse.tradista.ir.model" export="true" />
+		<module name="org.eclipse.tradista.mm.model" export="true" />
+		<module name="org.eclipse.tradista.security.model"
+			export="true" />
 		<module name="java.desktop" />
 		<module name="java.sql" />
 		<module name="java.naming" />

